src/Docfx.App/PdfBuilder.cs

```diff
@@ -641,4 +681,16 @@ private static StringComparison GetStringComparison()
             ? StringComparison.OrdinalIgnoreCase
             : StringComparison.Ordinal;
     }
+
+    private static CancellationTokenRegistration SubscribeCancelKeyPressEvent(CancellationTokenSource cts)
```

Comment:
This seems to apply to all build jobs, do we want to also enable it on the build command say in the future?

Reply:
I'm expecting same cancellation token should be passed to Build command also in future.